frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx

```diff
@@ -20,6 +20,8 @@ type ODFSystemDashboardPageProps = Omit<DashboardsPageProps, 'match'> & {
   match: Match<{ systemName: string }>;
 };
 
+const blockPoolRef = referenceForModel(CephBlockPoolModel);
```

Comment:
`referenceForModel` is deprecated , lets use `getGroupVersionKindForModel`

Reply:
`getGroupVersionKindForModel` will return an Object. That will not be useful here.

Reply:
Yes, one day you will need to change that , since only `getGroupVersionKindForModel` is exposed in sdk.